src/App.tsx

```diff
@@ -1,8 +1,78 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
 import './App.css';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoType, UserType, TodoWithUsers } from './types';
+import { getUsers, getTodos } from './api';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC<{}> = () => {
```

Comment:
в функциональных компонентах пустой объект не нужен
```suggestion
const App: FC = () => {
```

Reply:
Ок